script/tool/README.md

```diff
@@ -1,8 +1,60 @@
 # Flutter Plugin Tools
 
+Note: The commands in tools are designed to run under plugins/ or plugins/packages/.
+
 To run the tool:
 
 ```sh
 dart pub get
 dart run lib/src/main.dart <args>
```

Comment:
This command is misleading, given that as we've just said above this isn't the directory you should be in when running the tool.